src/components/Button.tsx

```diff
@@ -1,5 +1,5 @@
-import React from "react";
```

Comment:
Why is React import not at the top ?

Reply:
Used alt+shit+o for general rearrangement, it arranges in the alphabetical order.